DatadogRUM/Sources/RUMMonitor/Monitor.swift

```diff
@@ -464,6 +466,63 @@ extension Monitor: RUMMonitorProtocol {
         )
     }
 
+    // MARK: - Feature Operations
+
+    // swiftlint:disable function_default_parameter_at_end
+    func startFeatureOperation(name: String, operationKey: String?, attributes: [AttributeKey: AttributeValue]) {
```

Comment:
Should we have default values for `operationKey` (`nil`) and `attributes` (`[:]`)? This could also allow removing the linter exception.